index.html

```diff
@@ -49,5 +49,199 @@ <h2 class="job">I’am a Software Developer</h2>
+            /></a>
+          </li>
+          <li>
+            <a href=""
+              ><img class="html-opacity" src="image/ruby.svg" alt=""
+            /></a>
+          </li>
+        </ul>
+        <form action="index.html">
+          <button type="submit" class="projects-button">See Project</button>
+        </form>
+      </div>
+      <div class="profesional-react1">
+        <img class="prof1" src="image/pro.svg" alt="" />
+        <img class="text1" src="image/Supporting_ext.svg" alt="" />
+        <ul class="project-type">
+          <li>
+            <a href=""
+              ><img class="html-opacity" src="image/html.svg" alt=""
+            /></a>
+          </li>
+          <li>
+            <a href=""
+              ><img class="html-opacity" src="image/bootstrap.svg" alt=""
+            /></a>
+          </li>
+          <li>
+            <a href=""
+              ><img class="html-opacity" src="image/ruby.svg" alt=""
+            /></a>
+          </li>
+        </ul>
+        <form action="index.html">
+          <button type="submit" class="projects-button">See Project</button>
+        </form>
+      </div>
+    </div>
+    <div class="profesional-react1">
+      <img class="prof1" src="image/pro.svg" alt="" />
+      <img class="text1" src="image/Supporting_ext.svg" alt="" />
+      <ul class="project-type">
+        <li>
+          <a href=""
+            ><img class="html-opacity" src="image/html.svg" alt=""
+          /></a>
+        </li>
+        <li>
+          <a href=""
+            ><img class="html-opacity" src="image/bootstrap.svg" alt=""
+          /></a>
+        </li>
+        <li>
+          <a href=""
+            ><img class="html-opacity" src="image/ruby.svg" alt=""
+          /></a>
+        </li>
+      </ul>
+      <form action="index.html">
+        <button type="submit" class="projects-button">See Project</button>
+      </form>
+    </div>
+    <div class="aboutme-contener">
+      <div class="aboutme">
+        <img class="aboutme-img" src="image/aboutme.svg" alt="Aboutme" />
+        <p class="text-aboutme">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <form action="index.html">
+          <button type="submit" class="resume-button">Get My Resume</button>
+        </form>
+        <img class="line-image" src="image/line-image.svg" alt="" />
+        <div class="language">
+          <img src="image/Rectangle.svg" alt="" />
+          <img src="image/Language.svg" alt="" />
+        </div>
+        <ul class="languge-list">
+          <li class="languge-list-item1">Javascript</li>
+          <li class="languge-list-item2">Ruby</li>
+          <li class="languge-list-item3">Html</li>
+          <li class="languge-list-item4">Css</li>
+        </ul>
+
```

Comment:
## Changes required
⚠️ When I open your project, the fonts for your languages are not the ones used in the design. Please kindly fix them!

![image](https://user-images.githubusercontent.com/53036416/164836926-5314ce0a-5952-4760-b212-a05ca120282d.png)